packages/keyring/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@alephium/keyring",
+  "version": "0.0.0",
+  "private": true,
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "type": "module",
+  "scripts": {
+    "build": "tsup",
+    "watch": "tsup --watch",
+    "lint": "eslint . --ext .ts --max-warnings=0",
+    "lint:fix": "eslint . --fix --ext .ts",
+    "test": "jest --useStderr --silent=false --verbose=true --config jestconfig.json --passWithNoTests",
+    "test:watch": "pnpm run test -- --watch",
+    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\" --log-level silent",
+    "clean": "rm -rf dist",
+    "clean:node-modules": "rm -rf node_modules",
+    "typecheck": "tsup"
+  },
+  "dependencies": {
+    "@alephium/shared": "workspace:*",
+    "@alephium/shared-crypto": "workspace:*",
+    "@alephium/web3": "0.29.1",
+    "@metamask/scure-bip39": "^2.0.3",
+    "blakejs": "^1.1.1",
+    "ethereum-cryptography": "^2.1.2"
```

Comment:
To eliminate all possibilities I used the same package as Metamask extension does: 
- https://github.com/MetaMask/metamask-extension/blob/v11.13.1/package.json#L292
- https://github.com/MetaMask/eth-hd-keyring/blob/v7.0.1/package.json#L32